src/shared/components/palette/utils/first-error.pipe.ts

```diff
@@ -6,24 +6,28 @@ import { ValidationErrors } from '@angular/forms';
 })
 export class FirstErrorPipe implements PipeTransform {
 
-  transform(value: ValidationErrors): string {
+  transform(value: ValidationErrors, args?: string): string {
     if (!value) {
       return '';
     }
 
+    if (!args) {
+      args = 'field';
+    }
+
     let keys = Object.keys(value);
 
     if (!keys.length) {
       return '';
     }
     if (keys[0] ===  'required') {
-      return 'This field is required';
+      return `${args} is required`;
     } else if (keys[0] ===  'pattern') {
-      return 'The data entered is not valid for this type of field.';
+      return `The data entered is not valid for ${args}`;
     } else if (keys[0] ===  'minlength') {
-      return 'Required minimum length';
+      return `${args} required minimum length`;
```

Comment:
Same as my comment above about the phrasing. Why is this one in the past tense? 🤷‍♂️

Reply:
Changed as mentioned